src/DataReplication/custom/Start-AzDataReplicationUnplannedFailoverJob.ps1

```diff
@@ -0,0 +1,106 @@
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+function Start-AzDataReplicationUnplannedFailoverJob {
+    [CmdletBinding(DefaultParameterSetName = 'ByProtectedItemId', PositionalBinding = $false, ConfirmImpact = 'Medium')]
+    param (
+
+        [Parameter(ParameterSetName = 'ByProtectedItemId', Mandatory)]
+        [Parameter(ParameterSetName = 'ByProtectedItemIdExpanded', Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Path')]
+        [System.String]
+        # Specifies the Protected Item Id
+        ${ProtectedItemId},
+
+        [Parameter(ParameterSetName = 'ByInputObject', Mandatory)]
+        [Parameter(ParameterSetName = 'ByInputObjectExpanded', Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Models.Api20210216Preview.IProtectedItemModel]
+        # Specifies Object of type ProtectedItemModel
+        ${InputObject},
+
+        [Parameter(ParameterSetName = 'ByInputObject', Mandatory)]
+        [Parameter(ParameterSetName = 'ByProtectedItemId', Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Models.Api20210216Preview.IUnplannedFailoverModelCustomProperties]
+        ${CustomProperty},
+
+        [Parameter(ParameterSetName = 'ByInputObjectExpanded', Mandatory)]
+        [Parameter(ParameterSetName = 'ByProtectedItemIdExpanded', Mandatory)]
+        [ValidateSet("VMwareToAvs","VMwareToAvsFailback")]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Body')]
+        [System.String]
+        # Specifies instance type
+        ${InstanceType},
+
+        [Parameter(ParameterSetName = 'ByInputObjectExpanded')]
+        [Parameter(ParameterSetName = 'ByProtectedItemIdExpanded')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Body')]
+        [System.Boolean]
+        # Specifies whether to shut down the VM after failover
+        ${PerformShutdown},
+
+        [Parameter(ParameterSetName = 'ByInputObjectExpanded')]
+        [Parameter(ParameterSetName = 'ByProtectedItemIdExpanded')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Body')]
+        [System.String]
+        # Specifies name of recovery point
+        ${RecoveryPointName},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataReplication.Runtime.DefaultInfo(Script = '(Get-AzContext).Subscription.Id')]
+        [System.String]
+        # Specifies the subscription id.
+        ${SubscriptionId}
+    )
+
+    process {
+
+        $parameterSet = $PSCmdlet.ParameterSetName
+        $acceptedInstanceTypes = "VMwareToAvs","VMwareToAvsFailback"
```

Comment:
We support unplanned failover from on premise VMware to AVS. Remove VMwareToAvsFailback